Source/API/EbApi.h

```diff
@@ -151,7 +151,10 @@ extern "C" {
         // Quantization
         uint32_t                qp;
         uint8_t                 useQpFile;
-
+#if 1//TILES
```

Comment:
you meant "#if TILES"?

Reply:
yes, #define TILES , is defined in definitions.h , and it is not visible in the app, so I left it ON, but commented out to be able to track all changes when searching for TILES

Reply:
"#if 1//TILES" is not necessary and confusing. "#if TILES" not necessary either. A internal parameter can be defined to enable/disable tile based on TileRowCount/TileColumnCount value.
e.g. if both TileRowCount/TileColumnCount > 1, tile is enabled.